src/introduction.tex

```diff
@@ -7,59 +7,58 @@ \chapter{Introduction}
 technologies that orchestrate entire clusters, providing automatization that
 makes managing such systems easier.
 
-\textit{Kubernetes} is an open-source container orchestrator developed by
+\textbf{Kubernetes} is an open-source container orchestrator developed by
 Google. It is used by leading tech companies to automate the deployment,
 scaling, and management of their applications~\cite{kube-usecase}. Its common
 use cases are cloud computing systems and microservices-oriented architectures.
 
-The main feature provided by \textit{Kubernetes} and similar orchestration
+The main feature provided by Kubernetes and similar orchestration
 systems is the management of containerized applications. Containers are
 a~lightweight and portable alternative to virtual machines. In recent years,
 the~advantages of containerization have been recognised, which has led to
 the~rapid development of supporting technologies, such as Docker or rkt.
 
 A central part of many web applications is a~performant database. Despite
 the~variety of new database engines constantly appearing on the market, many
-developers still decide to rely on \textit{MySQL}. According to the db-engines
-statistics~\cite{db-eng}, \textit{MySQL} is the second most used database
+developers still decide to rely on \textbf{MySQL}. According to the db-engines
+statistics~\cite{db-eng}, MySQL is the second most used database
 engine.
 
-Our project is focused on developing tools for creating and managing
+Our project is focused on developing tools for setting up and managing
 a~\textbf{LAMP}\footnote{Linux host with Apache web server, MySQL database
-and PHP programming language} stack in \textit{Kubernetes}. Our main goal is to
-extend the \textit{Kubernetes API} with options for \textit{MySQL} database
+and PHP programming language} stack in Kubernetes. Our main goal is to
+extend the Kubernetes API with options for MySQL database
 management and provide users with a user-friendly command line tool. As
 a~result, developers should be able to more effortlessly benefit from
-\textit{Kubernetes’} advantages, such as scalability. Leveraging the powerful
-abstraction system provided by \textit{Kubernetes} in such a way will remove
+Kubernetes’ advantages, such as scalability. Leveraging the powerful
+abstraction system provided by Kubernetes in such a way will remove
 much of the complexity traditionally caused by the database layer of
 an~application.
 
 There are existing unofficial tools with similar functionalities for various
-databases and systems. \textit{Vitess}~\cite{vitess} is an~open source
-clustering system for \textit{MySQL} databases that can interface with
-\textit{Kubernetes}. We are aiming for a simpler solution that utilizes
-\textit{Kubernetes'} native method of extending its API ---
+databases and systems. Vitess~\cite{vitess} is an~open source
```

Comment:
Don't we want to add \textbf{vitess} here? It's a definition, right?

Reply:
We never mention it again in the thesis though, so it is a definition by definition, but not a definition we really care about.